### PR TITLE
Issue#18 implement follow up visit buttons on client profile

### PR DESCRIPTION
**Pull request by zzainulabidin**

### CODE REVIEW

File: opensrp-fp/src/main/java/org/smartregister/fp/features/profile/view/ProfileOverviewFragment.java [hunk 1]
```diff
@@ -57,21 +59,26 @@ protected void onResumption() {
             if (getActivity() != null && getActivity().getIntent() != null) {
                 HashMap<String, String> clientDetails =
                         (HashMap<String, String>) getActivity().getIntent().getSerializableExtra(ConstantsUtils.IntentKeyUtils.CLIENT_MAP);
+                baseEntityId = getActivity().getIntent().getStringExtra(ConstantsUtils.IntentKeyUtils.BASE_ENTITY_ID);
                 if (clientDetails != null) {
-                    contactNo = Utils.getTodayContact(clientDetails.get(DBConstantsUtils.KeyUtils.NEXT_CONTACT));
+                    List<HashMap<String, String>> data = FPLibrary.getInstance().getPreviousContactRepository().getVisitHistory(baseEntityId);
+//                    contactNo = Utils.getTodayContact(clientDetails.get(DBConstantsUtils.KeyUtils.NEXT_CONTACT));
```
Comment (rkareko) [inline opensrp-fp/src/main/java/org/smartregister/fp/features/profile/view/ProfileOverviewFragment.java]: Please remove unused code

File: reference-app/src/main/java/org/smartregister/sample/fp/app/FPApplication.java [hunk 2]
```diff
@@ -168,4 +168,10 @@ public void logoutCurrentUser() {
         startActivity(intent);
         context.userService().logoutSession();
     }
+
+    /*@NonNull
+    @Override
+    public ClientProcessorForJava getClientProcessor() {
+        return BaseFPClientProcessorForJava.getInstance(this);
+    }*/
```
Comment (rkareko) [inline reference-app/src/main/java/org/smartregister/sample/fp/app/FPApplication.java]: Please remove unused code

Comment (zzainulabidin) [reply reference-app/src/main/java/org/smartregister/sample/fp/app/FPApplication.java]: Removed

File: opensrp-fp/src/main/java/org/smartregister/fp/features/profile/view/ProfileOverviewFragment.java (hunk 1 above)
Comment (zzainulabidin) [reply opensrp-fp/src/main/java/org/smartregister/fp/features/profile/view/ProfileOverviewFragment.java]: Removed

**Comment by zzainulabidin** [comment]
https://github.com/OpenSRP/opensrp-client-fp/issues/18

**Review by Rkareko — APPROVED** [review]

**Comment by zzainulabidin** [comment]
https://github.com/OpenSRP/opensrp-client-fp/issues/18

**Pull request merged**